wm/config_theme_setters.go

```diff
@@ -160,23 +166,31 @@ func setGradient(k wini.Key, clr *render.Color) {
 }
 
 func setImage(k wini.Key, place **xgraphics.Image) {
-	if v, ok := getLastString(k); ok {
-		img, err := xgraphics.NewFileName(X, v)
-		if err != nil {
-			logger.Warning.Printf(
-				"Could not load '%s' as a png image because: %v", v, err)
-			return
-		}
-		*place = img
+	v, ok := getLastString(k)
+	if !ok {
+		return
+	}
+	b, err := misc.DataFile(v)
+	if err != nil {
+		logger.Warning.Printf(
+			"Could not load '%s' as a png image because: %v", v, err)
+	}
+
+	img, _, err := image.Decode(bytes.NewBuffer(b))
+	if err != nil {
+		logger.Warning.Printf(
+			"Could not load '%s' as a png image because: %v", v, err)
 	}
+	*place = xgraphics.NewConvert(X, img)
 }
 
 func setFont(k wini.Key, place **truetype.Font) {
 	if v, ok := getLastString(k); ok {
-		bs, err := ioutil.ReadFile(v)
+		bs, err := misc.DataFile(v)
 		if err != nil {
 			logger.Warning.Printf(
 				"Could not get font data from '%s' because: %v", v, err)
+			panic("font")
```

Comment:
I saw the related reviews about `panic` and wanted to point one more out here but if you left it in intentionally don't worry about it.

Reply:
Fixed, thanks.